api/clients/v2/accountant.go

```diff
@@ -1,113 +1,71 @@
 package clients
 
 import (
-	"errors"
+	"context"
 	"fmt"
 	"math/big"
 	"sync"
-	"time"
 
 	"github.com/Layr-Labs/eigenda/core"
 	"github.com/Layr-Labs/eigenda/core/meterer"
 	gethcommon "github.com/ethereum/go-ethereum/common"
 )
 
 type Accountant struct {
-	// on-chain states
-	accountID    gethcommon.Address
-	reservations map[core.QuorumID]*core.ReservedPayment
-	// OnDemand is initially only enabled on quorum 0. Accountant must be updated to be quorum specific
-	// after the protocol decides to support onDemand on custom quorums and decentralized ratelimiting.
-	onDemand *core.OnDemandPayment
+	// account identification
```

Comment:
I'm curious why you are wrapping the new `AccountLedger` inside of the accountant. 

The original idea was for the `AccountLedger` to replace the accountant. If we just leave the current `Accountant` as-is for now, then we are able develop the new payment classes without having any impact on the current system. And when we're ready, we can just switch over.

Reply:
was eager to test out the functionality of LocalAccountLedger; I've now reverted leave `Accountant` as-is for now [a5581f8](https://github.com/Layr-Labs/eigenda/pull/1694/commits/a5581f871327b47cc67b1a5d670962a9dcc4fe3f)